### PR TITLE
chermaine-notes.txt

### PR DESCRIPTION
**Pull request by 19chermaine98**

### CODE REVIEW

**Comment by crazoter** [comment]
Ayyyyy

**Comment by madanalogy** [comment]
WOW GIT GUD SCRUB

**Comment by LiuZechu** [comment]
hi

File: chermaine-notes.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+git init
+	intialize a new git repository
+
+git status
+	tells us the status of our working directory
+	red = changes are in cardboard box
+	green = changes are in wooden box
+	
+	workign tree clean = changes are in repository (metal box)
```
Comment (yyc) [inline chermaine-notes.txt]: workign => working

**Review by yyc — CHANGES_REQUESTED** [review]
fix typo pls

File: chermaine-notes.txt (hunk 1 above)
Comment (madanalogy) [reply chermaine-notes.txt]: How you link to the file Sia

**Pull request merged**